### PR TITLE
Improve the documentation and overall structure of the project

### PR DESCRIPTION
**Pull request by vedran-kasalica**
Changes align the project with the software development [best practices](https://www.bestpractices.dev/en/projects/8083)

- [x] Improve the library documentation
- [x] Introduce guidelines
- [ ] Improve CI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ A package ([documentation](https://biomarkersparkinson.github.io/tsdf/)) to load
 
 ### Using `pip`
 
-The package is available in PyPi. It can be installed using:
+The package is available in PyPi and requires [Python 3.9](https://www.python.org/downloads/) or higher. It can be installed using:
```
Comment (kretep) [inline README.md:20]: Any specific reason for this requirement?

Comment (vedran-kasalica) [reply README.md:20]: https://github.com/biomarkersParkinson/tsdf/blob/8d44032b6f3290814d9bd1c5ccdb391256a8cd11/pyproject.toml#L19 also see #46

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- Steps to reproduce the vulnerability.
+- Versions of tsdf affected.
+
+We are committed to responding promptly to security reports. We will acknowledge your email within 48 hours and will strive to provide a timeline for resolution as soon as possible.
+
+## Security Response Team
+
+Our security response team consists of the following individuals:
+
+- Vedran Kasalica (p.kok[at]esciencecenter.nl)
```
[the remark this answers is not in the document]
Comment (vedran-kasalica) [reply SECURITY.md]: Thanks, I just fixed that

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+If you discover a security vulnerability in tsdf, please report it by sending an email to p.kok[at]esciencecenter.nl. We ask that you do not publicly disclose the issue until it has been addressed.
+
+In your report, please provide the following information:
+
+- A brief description of the vulnerability.
+- Steps to reproduce the vulnerability.
+- Versions of tsdf affected.
+
+We are committed to responding promptly to security reports. We will acknowledge your email within 48 hours and will strive to provide a timeline for resolution as soon as possible.
```
Comment (kretep) [inline SECURITY.md:22]: I'm not sure if we can promise this after the end of the project, as it's not certain there will be budget for that. Also, if we promise 48 hours, there should be more people in the team, taking into account holidays and such.

**Review by kretep — APPROVED** [review]

**Pull request merged**